source/getting_started.md

```diff
@@ -1365,7 +1365,7 @@ class Article < ApplicationRecord
 
   VALID_STATUSES = ['public', 'private', 'archived']
 
-  validates :status, in: VALID_STATUSES
+  validates :status, inclusion: VALID_STATUSES
```

Comment:
should be

```suggestion
  validates :status, inclusion: { in: VALID_STATUSES }
```

https://edgeguides.rubyonrails.org/getting_started.html#using-concerns